src/AdScreenerTool.js

```diff
@@ -240,13 +245,52 @@ const AdScreener = (params) => {
   	setSelectedTopicOrFullTextSearchTab(k);
   	if(k=='topics') {
   		setFullTextSearchQuery(null);
+		setPageId(null);
   	}
+	else if(k=='advertiser') {
+		setFullTextSearchQuery(null);
+		setTopic({ selectedOption: "" });
+		setTopicParam(undefined);
+	}
   	else {
-		setTopic({ selectedOption: ""});
+		setTopic({ selectedOption: "" });
 		setTopicParam(undefined);
+		setPageId(null);
   	}
   }
 
+  const [isLoading, setIsLoading] = useState(false);
+  const [options, setOptions] = useState([]);
```

Comment:
These are specific to advertiser search, right? If so, please indicate that in the variables names. There are multiple things in this context that could be loading or have options.